examples/2d/bar_waves/step01_pwave_reflected.cfg

```diff
@@ -80,7 +80,7 @@ db_auxiliary_field.label = Dirichlet BC on domain
 
 [pylithapp.problem.bc.bc_domain]
 constrained_dof = [1]
-label = domain_all
+label = marker 
```

Comment:
This name might be confusing since it is usually applied to the entire domain boundary.